src/test/kotlin/com/ajax/cryptocurrency/redis/RedisCryptocurrencyRepositoryTest.kt

```diff
@@ -0,0 +1,101 @@
+package com.ajax.cryptocurrency.redis
+
+import com.ajax.cryptocurrency.config.TestConfig
+import com.ajax.cryptocurrency.domain.DomainCryptocurrency
+import com.ajax.cryptocurrency.infrastructure.database.mongo.entity.CryptocurrencyEntity
+import com.ajax.cryptocurrency.infrastructure.database.redis.RedisCryptocurrencyRepository
+import com.ajax.cryptocurrency.infrastructure.mapper.CryptocurrencyMapper
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.bson.types.ObjectId
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.data.redis.core.ReactiveRedisTemplate
+import org.springframework.test.context.ContextConfiguration
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+import reactor.test.StepVerifier
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+
+@SpringBootTest
+@ContextConfiguration(classes = [TestConfig::class])
+class RedisCryptocurrencyRepositoryTest {
```

Comment:
It's mix of Unit and Integration test.
Let's choose on of them

Reply:
I've redesigned the tests so that there is no mix of unit and integration tests